modules/gitops-configuring-a-cluster-scoped-instance-to-manage-rollouts-resources.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-cluster-scoped-rollouts-instance-to-manage-rollouts-resources.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-a-cluster-scoped-instance-to-manage-rollouts-resources_{context}"]
+= Configuring a cluster-scoped rollouts instance to manage rollouts resources 
+
+To configure a cluster-scoped rollouts instance for managing rollouts resources, add the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable in the `Subscription` resource. This variable contains a list of user-defined namespaces which can be configured for a cluster-scoped rollouts installation.
+
+[NOTE]
+====
+You can only create a cluster-scoped rollouts instance if the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` variable is set to `false`.
+====
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* → *Installed Operators* → *{gitops-title}* → *Subscription*.
+
+. Click the *Actions* list and then click *Edit Subscription*.
+
+. On the *openshift-gitops-operator* Subscription details page, under the *YAML* tab, edit the `Subscription` YAML file by adding the namespace of the Argo CD instance to the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable in the `spec` section:
++
+.Example: Configuring the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: argo-operator
+spec:
+  config:
+   env: 
+    - name: NAMESPACE_SCOPED_ARGO_ROLLOUTS
+      value: 'false' #<1>
+    - name: CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES
+      value: <list_of_namespaces_in_the_cluster-scoped_Argo_CD_instances> #<2>
+ ...
+----
+<1> The value set to `'false'` means that the installation defaults to cluster-scoped mode.
```

Comment:
Seems misleading, it doesnt default to cluster-scoped, but it allows to create cluster-scoped instance, if all other configurations are done properly, but it is set to true then cluster-scoped installation will not be allowed. It is a switch to enable/disable cluster-scoped installation.

Reply:
Fixed. PTAL.